raiden_libs/network/matrix/client.py

```diff
@@ -35,6 +36,7 @@ def __init__(
             pool_maxsize: int = 10,
             retry_timeout: int = 60,
             retry_delay: Callable[[], Iterable[float]] = lambda: repeat(1),
+            long_paths: Container[str] = tuple(),
```

Comment:
This default argument should be probably `None`; and instantiate `tuple()` if needed at L54

Reply:
Why? The idea is exactly that the default is no _priority_ paths, so empty is as expressive, and avoid the additional instantiation. `None` has no different/special meaning beyond an empty tuple, to be needed.

Reply:
Reason being [mutable default arguments](https://docs.python-guide.org/writing/gotchas/). It could cause some flakyness/inconsistency in the tests.

Reply:
`tuple` isn't mutable ;)